components/frontend/src/issue/IssuesRows.js

```diff
@@ -68,19 +68,19 @@ function IssueIdentifiers({ entityKey, issueTrackerInstruction, metric, metric_u
     )
 }
 IssueIdentifiers.propTypes = {
-    entityKey: PropTypes.string,
-    issueTrackerInstruction: PropTypes.node,
+    entityKey: string,
+    issueTrackerInstruction: node,
     metric: metricPropType,
-    metric_uuid: PropTypes.string,
-    report_uuid: PropTypes.string,
-    target: PropTypes.string,
-    reload: PropTypes.func
+    metric_uuid: string,
+    report_uuid: string,
+    target: string,
+    reload: func
 }
 
 export function IssuesRows({ metric, metric_uuid, reload, report, target }) {
     const parameters = report?.issue_tracker?.parameters;
     const issueTrackerConfigured = Boolean(report?.issue_tracker?.type && parameters?.url && parameters?.project_key && parameters?.issue_type);
-    const issueTrackerInstruction = issueTrackerConfigured ? null : <p>Please configure an issue tracker by expanding the report title, selecting the 'Issue tracker' tab, and configuring an issue tracker.</p>;
+    const issueTrackerInstruction = issueTrackerConfigured ? null : <p>Please configure an issue tracker by expanding the report title, selecting the &apos;Issue tracker&apos; tab, and configuring an issue tracker.</p>;
```

Comment:
Interesting that this string literal doesn't need to be quoted, is it not something worth adding explicitly?

Reply:
It's not a string literal, it's JSX I guess.